website/docs/patterns/button-organization/partials/guidelines/alignment.md

```diff
@@ -6,14 +6,14 @@ Alignment refers to how buttons and actions are aligned relative to the containe
 
 When considering internationalization (i18n), **start** and **end** are suitable substitutes for **left** and **right** alignment to account for languages that are read right-to-left ([RTL](https://developer.mozilla.org/en-US/docs/Glossary/RTL)).
 
-![Button alignment options](/assets/patterns/button-organization/alignment-methods.png)
+![Button alignment options](/assets/patterns/button-organization/alignment-methods.png =735x*)
```

Comment:
Have you considered to use the same width for all the images, as suggested here?
https://hashicorp.slack.com/archives/C025N5V4PFZ/p1688633989191419

It's much easier to do (and maintain) than manually adding a size to all the images.

Reply:
Thanks for the reminder of that, I updated the all of the image sizes and they should all be consistent

Reply:
I just looked through the images, and they still show inconsistent sizes (e.g., 1380, 1470, 1540, 1672). I believe the idea is that all images would be the same size. Also, if going this route, you'll need to look through the images in the markdown and remove the size value (some of them still have them).